include/cantera/kinetics/Reaction.h

```diff
@@ -249,6 +249,48 @@ class ElectrochemicalReaction : public InterfaceReaction
     bool exchange_current_density_formulation;
 };
 
+//! A reaction with rate parameters for Blowers Masel approximation
+class BlowersMaselReaction: public Reaction
+{
+public:
+    BlowersMaselReaction();
+    BlowersMaselReaction(const Composition& reactants,
+                         const Composition& products, const BlowersMasel& rate);
+    virtual void validate();
+    BlowersMasel rate;
+    bool allow_negative_pre_exponential_factor;
+};
+
+//! A reaction occurring on an interface (i.e. a SurfPhase or an EdgePhase)
+//! with the rate calcualted with Blowers Masel approximation.
```

Comment:
```suggestion
//! with the rate calculated with Blowers Masel approximation.
```
Also, I'm assuming this should be `Blowers-Masel` throughout the documentation?